src/str/adapters.rs

```diff
@@ -111,6 +112,27 @@ where
     fn from_utf8(self) -> Utf8Predicate<Self> {
         Utf8Predicate { p: self }
     }
+
+    /// Returns a `NormalizedPredicate` that ensures
+    ///  the newlines within the data passed to `Self` is normalised.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
Still not seeing this example in the test results
https://travis-ci.org/assert-rs/predicates-rs/jobs/408337538

Is it because its missing "rust" on the code fence?